pihole

```diff
@@ -62,6 +62,12 @@ function setupLCDFunction {
     exit 1
 }
 
+function queryFunc {
+    domain=$2
+    for list in /etc/pihole/list.*;do echo $list;grep ${domain} $list;done
+    exit 1
```

Comment:
Just one of the debug commands we run a lot to find what list contains the domain in question. Is there something you see off in there?